README.md

```diff
@@ -142,6 +142,28 @@ Actions are a series of functions to run. Availabe actions are:
           Content-Type: text/html
 ```
 
+The actions by default run in the order defined in the mock file; you can adjust this by adding an int 'order' field.
+```yaml
+- key: every-op
+  kind: Behavior
+  expect:
+    http:
+      method: GET
+      path: /ping
+  actions:
+    - publish_kafka:
+        topic: hello_kafka_out
+        payload: >
+          {
+            "kafka": "OK",
+            "data": {}
+          }
+    - sleep:
+        duration: 1s
+      # sleep first
+      order: -1
```

Comment:
I like the order field, very neat. One comment about having `-1` is that in a lot of languages it means the last one, can we do something different here?

Reply:
option 1: if I set this value `-1000` is that clearer? 
option 2: should we only allow 0 and positive integers for order?  my thinking was it's conceptually difficult to change the order of any actions you inherited with extends, so it makes more sense to let users go before the 'default' value.   Maybe if we use positive integers the default value should be 100.

Reply:
pushed a version with the examples showing `-1000` for clarity